Tasks/Xcode/xcode.ts

```diff
@@ -208,21 +202,10 @@ async function run() {
         var publishResults: boolean = tl.getBoolInput('publishJUnitResults', false);
 
         if (publishResults) {
-            if (useXctool) {
-                if (xctoolReporter && 0 !== xctoolReporter.length) {
-                    var xctoolReporterString = xctoolReporter.split(":");
-                    if (xctoolReporterString && xctoolReporterString.length === 2) {
-                        testResultsFiles = tl.resolve(workingDir, xctoolReporterString[1].trim());
-                    }
-                } else {
-                    tl.warning(tl.loc('UseXcToolForTestPublishing'))
-                }
-            } else if (!useXctool) {
-                if (!useXcpretty) {
-                    tl.warning(tl.loc('UseXcprettyForTestPublishing'));
-                } else {
-                    testResultsFiles = tl.resolve(workingDir, '**/build/reports/junit.xml');
-                }
+            if (!useXcpretty) {
+                tl.warning(tl.loc('UseXcprettyForTestPublishing'));
+            } else {
+                testResultsFiles = tl.resolve(workingDir, '**/build/reports/junit.xml');
             }
 
             if (testResultsFiles && 0 !== testResultsFiles.length) {
```

Comment:
The package section has to be updated to remove usage of xcrun from the implementation